### PR TITLE
fixing broken link

### PR DESCRIPTION
**Pull request by danieldobalian**

### CODE REVIEW

**Comment by danieldobalian** [comment]
Addressing #246

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Copyright (c) Microsoft Corporation.  All rights reserved. Licensed under the MI
 
 ### Requirements
 * Android SDK 21+
-* [Chrome](https://github.com/Azure-Samples/active-directory-general-docs/blob/master/AndroidEmulator.md) 
+* [Chrome Custom Tabs](https://github.com/Azure-Samples/active-directory-general-docs/blob/master/AndroidEmulator.md)
```
Comment (iambmelt) [inline README.md]: Strictly speaking, there is [fallback logic to support users who do not have `CustomTabs` but _do_ have Chrome](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/client/AuthenticationActivity.java#L234-L238). I think this text is OK to leave as 'Chrome'

File: README.md [hunk 2]
```diff
@@ -176,6 +176,6 @@ Copyright (c) Microsoft Corporation.  All rights reserved. Licensed under the MI
 
 ### Step 4: Use the token!
 
-The access token can now be used in an [HTTP Bearer request](https://github.com/Azure-Samples/active-directory-android-native-v2/blob/master/app/src/main/java/com/danieldobalian/msalandroidapp/MainActivity.java#L152).
+The access token can now be used in an [HTTP Bearer request](https://github.com/Azure-Samples/active-directory-android-native-v2/blob/56b8cca297adf616946e9ac2200847b71fd7578a/app/src/main/java/com/azuresamples/msalandroidapp/MainActivity.java#L152).
```
Comment (iambmelt) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (danieldobalian) [reply README.md]: Reverted.

**Pull request merged**